### PR TITLE
Translation-ja: Add trusted-committer/ja/01-introduction-ja

### PR DESCRIPTION
**Pull request by ystk**
This is the first PR for #367 .

### CODE REVIEW

**Comment by lenucksi** [comment]
Hey @ystk! Thanks a lot for translating more of the learing path - I'm curious to see what the article that I was a bit more active on writing than with the others will look like 😄 🎉 
As always: I'm happy to merge once we have another review from another Japanese-speaking person. Could we maybe interest you for this again, @johtani?

File: trusted-committer/ja/01-introduction-ja.asciidoc
Comment (johtani) [inline trusted-committer/ja/01-introduction-ja.asciidoc:40]: 「経営陣」という言葉が少ししっくりこなかったのですが、代わりのいい言葉も浮かんでこなかったです。introductionの02では「マネージメント層」という役だったので「マネージメント」でもいいのかもしれません。

**Review by johtani — COMMENTED** [review]
Sorry for the late reply. 
I left some comments.

**Comment by rrrutledge** [comment]
@johtani you're doing a great service in these reviews!

**Comment by ystk** [comment]
@johtani Thanks for your suggestions! I updated the file. Please check it.

**Review by johtani — APPROVED** [review]
LGTM!

**Review by rrrutledge — APPROVED** [review]
Woo hoo!  Thank you!

**Pull request merged**
